src/pages/Staking/index.tsx

```diff
@@ -665,7 +682,9 @@ const Staking = (): JSX.Element => {
   const renderEstimatedAPYLabel = () => {
     if (
       estimatedRewardAmountAndAPYIdle ||
-      estimatedRewardAmountAndAPYLoading
+      estimatedRewardAmountAndAPYLoading ||
+      errors[LOCK_TIME] ||
+      errors[LOCKING_AMOUNT]
```

Comment:
Similarly

If `LOCK_TIME` or `LOCKING_AMOUNT` has a validation error, just displaying `-` instead of a meaningless value makes more sense from a UX perspective.